docs/development/i18n.rst

```diff
@@ -92,43 +100,58 @@ new language is completely translated and reviewed, the
 ``i18n_tool.py`` file must be manually edited to add this new language
 to the ``SUPPORTED_LANGUAGES`` variable.
 
+.. _update_strings_to_be_translated:
+
 Update strings to be translated
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-After strings are modified in the SecureDrop source code, templates or
-desktop labels, the ``securedrop/translations/messages.pot`` files
-must also be updated. Individual developers should not do this
-whenever changing strings in the code; the translations are updated in
-bulk when it's time to update the Weblate fork.
-
-Translations can be updated with the following command:
-
-.. code:: sh
-
-    $ make translate
-
-This wraps ``i18n_tool.py translate-messages`` and ``i18n_tool.py
-translate-desktop``.  These commands update the `.pot`_ files for the
-SecureDrop server code and the desktop icons, as well as the `.po`_
-files for each language.
-
-.. note:: The new source strings will only be visible to translators
-    in `Weblate`_ after the ``develop`` branch is merged into the
-    Weblate fork.
+Whenever strings are modified in the SecureDrop source code, templates
```

Comment:
could you link to an example of a string in a template that would require translations? or maybe just share here. i'm confused about which code you're talking about that could require translations outside of the app and app desktop icon.

Reply:
Any string in an HTML template marked for translation with one of the `gettext` functions will require the translation files to be updated. Is the text any clearer now?

Reply:
i see, and yes, the text is clearer. thx!